endaq/device/command_interfaces.py

```diff
@@ -2086,13 +2105,67 @@ def reset(self,
             :returns: `True` if the command was successful.
         """
         # FUTURE: Write commands wrapped in a <EBMLCommand> element?
+        #  Exclude LegacyFileCommandInterface.
         return self._runSimpleCommand({'Reset': {}},
                                       timeoutMsg="Timed out waiting for device to reset",
                                       wait=wait,
                                       timeout=timeout,
                                       callback=callback)
 
 
+# ===========================================================================
+#
+# ===========================================================================
+
+class LegacyFileCommandInterface(FileCommandInterface):
```

Comment:
I don't see this instantiated anywhere outside of tests, am I missing something?

Reply:
There's a list of the `CommandInterface` subclasses at the bottom of the file. This gets iterated over, calling each class' `hasInterface()` method, and the first one that returns `True` gets instantiated.